src/ci_tools.ts

```diff
@@ -53,6 +56,18 @@ async function run(originalArgv: string[]): Promise<void> {
     printHelp();
     process.exit(1);
   }
+
+  const mode: string = argv.mode || DEFAULT_MODE;
```

Comment:
Ich würde auch gerne noch einmal für TypeInference plädieren:

```suggestion
  const mode = argv.mode || DEFAULT_MODE;
```